coralproject/talk
# [CORL-2510]: Upgrade eslint to v8

## What does this PR do?

Upgrades `eslint` to v8. Also upgrades related packages. This also includes removing linting from watch and just letting linting be run separately and as a hook when committing files.

For linting issues that came up, either types were added in or eslint was disabled for the next line.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Run `npm-i.sh` to update packages. Then you can click around and see that everything works as expected, that Coral starts up as expected with no errors, etc.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
